infra/main.bicep

```diff
@@ -256,7 +257,7 @@ module searchService 'core/search/search-services.bicep' = {
     name: !empty(searchServiceName) ? searchServiceName : 'gptkb-${resourceToken}'
     location: !empty(searchServiceLocation) ? searchServiceLocation : location
     tags: tags
-    publicNetworkAccess: publicNetworkAccess == 'Enabled' ? 'enabled' : 'disabled'
+    publicNetworkAccess: toLower(publicNetworkAccess) == 'enabled' ? 'enabled' : 'disabled'
```

Comment:
I'm thinking this validation can be done in the search module itself, that ways all consumers don't have to.

Reply:
I see that search module allowed values are lower, wondering if we should add that here too?

Are you sharing the same publicNetworkAccess parameter for both search and cognitive?  Do we have a scenario that require them to be two settings?